common-npm-packages/webdeployment-common-v2/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "declaration": true,
+        "noImplicitAny": false,
+        "sourceMap": false
+    }
```

Comment:
I think we should exclude node_modules.

Reply:
@egor-bryzgalov I am not able understand this comment, can you please elaborate what needs to be done here

Reply:
add this to your tsconfig.json (e.g. after "sourceMap": false):
```
"exclude": [
  "node_modules"
]
```

Reply:
@egor-bryzgalov done